packages/workbox-background-sync/src/lib/request-manager.js

```diff
@@ -42,49 +42,58 @@ class RequestManager {
   }
 
   /**
+   * function to play one single request
+   * given its hash
+   * @param {String=} hash
```

Comment:
Are you using `{String=}` to indicate that this is an optional parameter? If so, we've been using `[hash]` elsewhere in the docs (the first option listed in http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values)